docs/source/development_js.rst

```diff
@@ -29,22 +29,36 @@ following steps are used when making a Notebook release.
 
          sudo apt-get install nodejs-legacy npm
 
-#. Build the JavaScript and CSS by running:
+#. Install the notebook:
 
+   In the notebook repo, do a development install:
+   
    .. code:: bash
+   
+       pip install -e .
 
-       python setup.py css js
+#. Rebuild JavaScript and CSS
 
-   This command will automatically fetch the remaining dependencies (bower,
+   There is a build step for the JavaScript and CSS in the notebook.
+   If you are modifying the JavaScript or LESS sources, you may need to rebuild
+   their targets from time to time. This can be done with:
```

Comment:
'from time to time' - doesn't it need to be any time you modify JS/CSS (assuming you want to test it)?